includes/class-sensei-settings-api.php

```diff
@@ -395,15 +395,22 @@ public function create_fields() {
 				$name   = $v['name'];
 				if ( $v['type'] == 'info' ) {
 					$name = ''; }
+
+				$class = '';
+				if ( ! empty( $v['unique_page'] ) ) {
+					$class = 'sensei-settings-unique-page';
+				}
```

Comment:
I had a small thought here, just wanted to share with you. Looks like the only utility of the `unique_page` property is to have a class added for it here. I was wondering if we should cut the middleman and directly enable adding a `class` property directly in the settings array. I mean, instead of `'unique_page' = true`, it'd be `'class' = 'sensei-settings-unique-page'`. That way, I think it will probably be more reusable, because any setting that wants a class can utilize that.

Reply:
Thanks, Imran. I get your point that using a class attribute is more reusable, but I prefer to have a concrete attribute for each feature - it highlights the intent. Maybe it will help to get my point across if you imagine multiple features that add a different class. That will make the intent less obvious and bloat the class attribute.

Please, let me know if you have a strong opinion about this. I'm open to discussions. 🙂